corehq/apps/style/static/dashboard/less/dashboard.less

```diff
@@ -52,6 +52,7 @@
   .panel-heading {
     font-weight: bold;
     font-size: @font-size-large;
+    text-align: center;
```

Comment:
An extra style isn't necessary unless we're going to have some dashboard .panel-heading elements that are center aligned and then others that are left-aligned...which doesn't make sense for consistency reasons.

In general, I'm not a fan of center aligning text, either - it's harder to read, and it breaks up visual lines on a page. But centered headers over centered content, like the dashboard buttons, is one of the places it can be appropriate.